src/System.Net.Sockets/tests/FunctionalTests/NetworkStreamTest.cs

```diff
@@ -623,6 +623,7 @@ public void Ctor_NotConnected_ThrowsIOException()
         [InlineData(4096)]
         [InlineData(4095)]
         [InlineData(1024*1024)]
+        [ActiveIssue(20135, TargetFrameworkMonikers.Uap)]
```

Comment:
Our current mission is to make all tests run clean for UWP. There is no known issue to track the problem, because I'm still investigating it. My following work for S.N.Sockets is to solve those ActiveIssue label: submit a fix or open a new issue.

cc: @davidsh